src/Core/Domain/Customer/ValueObject/ApeCode.php

```diff
@@ -59,11 +59,14 @@ public function getValue()
 
     private function assertIsApeCode($code)
     {
-        if (is_string($code) && empty($code)) {
+        if (!is_string($code)) {
+            throw new CustomerConstraintException(sprintf('Invalid ape code %s provided', var_export($code, true)), CustomerConstraintException::INVALID_APE_CODE);
+        }
```

Comment:
Too bad you're throwing twice the same exception, maybe you can find a way to throw it only once?

Reply:
Fixed